### ISSUE TITLE
Cross-check ICES Science Plan "Emerging Technologies" tasks/categories with DIG Challenges and Opportunities

### ISSUE THREAD

**Issue by davidcurrie2001** [ices-eg/DIG#600]
The ICES Science Plan specifies a number of tasks related to emerging technologies.  We should check if we have covered all of these within the DIG Challenges and Opportunities tracker.
- Horizon scan, test, develop and where appropriate harness new and emerging techniques and technologies that have potential to progress the ICES vision and mission; with an emphasis on data gathering, processing and interpretation. Emerging techniques and technologies 
- Develop more efficient ways of analysing, sharing and presenting big data from observation and monitoring; especially using data from remote sensing of the seas and monitoring of human activities. 
- Develop and apply a wide range of analytical and statistical tools, such as machine learning, to describe the state and dynamics of the marine environment and the distribution and dynamics of human activities, and assess their strengths and weaknesses. 
- Investigate the benefits and costs of techniques that may supplement or replace existing approaches to biological 'sampling', including the applications of acoustics, image analysis, molecular methods (e.g. eDNA, genetic barcoding and genetic close-kin mark-recapture method) as well as sensors for chemical and physical sampling. 
- Track the emergence of new technologies in marine industries and assess how these technologies affect the interactions between those industries and the marine environment.

**Comment by davidcurrie2001** [comment]
I cross-checked the Science and Advice Plans with the DIG Challenges and Opportunities – see [PlanCodes_20240327.xlsx](https://github.com/ices-eg/DIG/files/14787642/PlanCodes_20240327.xlsx).  A lot of the tasks aren’t within the scope of DIG so I didn’t consider them any further.  For the tasks that are in scope I identified any existing Challenges and Opportunities – after doing this there do seem to be a few things missing:  

**Potential missing Opportunities**
•	Develop efficient ways of analysing large ICES data sets
•	Operationalising new analysis techniques and data sources
•	Making ICES advice and data available in an interactive, web-based format
•	Making ICES data more easily available to the public
•	Incorporating new data submissions from industry in ICES

**Potential missing Challenges**
•	Managing potential or perceived conflicts of interest in data submissions

Do you think we should these into our tracker?  If we have consensus I can just add the new Challenges and Opportunities into our tracker.

**Comment by neil-ices-dk** [comment]
@davidcurrie2001 thanks for the good summary; i think we need to discuss each of these and be sure we have a common understanding of context before entering them in the DIG tracker as some of them are overlapping and perhaps we need further annotation. Is that something we should tackle online before DIG?

**Comment by davidcurrie2001** [comment]
Yes, it would be good to take a look before DIG.  I'll send around some meeting dates by email.

**Comment by davidcurrie2001** [comment]
We had an online meeting (10/5/24) and talked through the Advice and Science plan codes that seemed to be missing from the DIG Challenges & Opportunities (C&O) tracker.  We thought there could be a good case for including these.  It was agreed that the C&O sub-group during the DIG meeting should have the task of turning these candidate C&Os into entries on the tracker, as well as assigning them likelihood and impact scores.  This will be a good way of getting the wider DIG group involved in the process.
